rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.79"
+components = ["rustfmt", "clippy"]
```

Comment:
What is this file for?

Reply:
When you run cargo it will automatically make sure you have the right rust version installed. It's like .nvmrc files. Cargo interprets this file automatically